packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1345,6 +1393,8 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
 
         ReportOpPerfTelemetry(this.context.clientId, this.deltaManager, this.logger);
         BindBatchTracker(this, this.logger);
+
+        this.entryPoint = initializeEntryPoint?.(this);
```

Comment:
Went with delaying the call until entryPoint is accessed. Makes sense to keep the runtime's instantiation lean, especially since we don't know how complex that function could be for a particular container.